js/iframe-coordinator/Elm.d.ts

```diff
@@ -1,69 +1,58 @@
-declare module "*/Client.elm" {
-  export interface LabeledMsg {
-    msgType: string;
-    msg: any;
-  }
+interface LabeledMsg {
+  msgType: string;
+  msg: any;
+}
 
-  export interface MessageData {
-    origin: string;
-    data: any;
-  }
+interface SubscribeHandler {
+  (msg: LabeledMsg): void;
+}
 
-  export interface Worker {
-    requestToast: Function;
-    ports: {
-      fromHost: {
-        send: (message: MessageData) => void;
-      };
-      fromClient: {
-        send: (message: LabeledMsg) => void;
-      };
-      toHost: {
-        subscribe: (message: any) => void;
-      };
-      toClient: {
-        subscribe: (message: any) => void;
-      };
-    };
-  }
+interface InPort {
+  send(msg: LabeledMsg): void;
+}
 
-  interface Client {
-    worker(): Worker;
-  }
+interface OutPort {
+  subscribe(handler: SubscribeHandler): void;
+}
 
-  export var Client: Client;
+interface ClientProgram {
+  ports: {
+    fromHost: {
```

Comment:
Not applicable for this PR, but should we eventually switch these names to be consistent with the message types?